airflow/providers/google/cloud/operators/automl.py

```diff
@@ -47,12 +49,21 @@
     from airflow.utils.context import Context
 
 MetaData = Sequence[Tuple[str, str]]
+automl_nl_model_keys = [
+    "text_classification_model_metadata",
+    "text_extraction_model_metadata",
+    "text_sentiment_dataset_metadata",
+]
```

Comment:
Should we really keep this list outside of the class? It is used only in one place.

Reply:
Sure, thanks! i have updated file